docs/api/conformance/Basic-Tests.md

```diff
@@ -0,0 +1,33 @@
+---
+id: Basic-Tests
+sidebar_label: Basic Tests
+title: Basic Tests
+hide_title: true
+---
+
+# Basic Tests
+<!-- markdownlint-disable MD033 -->
+
+_These are some basic sanity tests implemented in the FDC3 Conformance Framework.  It is expected that Desktop Agent testers will run these first before commencing the much more thorough tests in section 2 onwards._
+
+- `BasicCL1`: You can create a context listener by calling `fdc3.addContextListener('fdc3.contact',<handler>)`.  A `Listener` object is returned and can be used to remove the listener again by calling its `unsubscribe` function.
+- `BasicCL2`: You can create an **unfiltered** context listener by calling `fdc3.addContextListener(null,<handler>)`.  A `Listener` object is returned and can be used to remove the listener again by calling its `unsubscribe` function.
```

Comment:
@kriswest Also we have some spaces after the first argument in the code samples. e.g. `fdc3.addContextListener(null,<handler>)` instead of `fdc3.addContextListener(null, <handler>)`, which are inconsistent with elsewhere e.g. App Channel Tests, Intents Test, and also _might_ have line wrapping implications if not viewed in a large-width window.

Reply:
removed all spaces in front of args and tried to standardize substitutes for context examples some (e.g. `<fdc3.instrument>` rather than `<an instrument context>`